metaci/cumulusci/views.py

```diff
@@ -17,14 +16,22 @@
 from metaci.cumulusci.models import Org
 from metaci.cumulusci.models import ScratchOrgInstance
 from metaci.cumulusci.utils import get_connected_app
+from metaci.plan.models import PlanRepository
 
 
-@staff_member_required
+@login_required
 def org_detail(request, org_id):
-    org = get_object_or_404(Org, id=org_id)
+    try:
+        org = Org.objects.for_user(request.user).get(id=org_id)
+    except Org.DoesNotExist:
+        raise Http404
```

Comment:
Maybe we'll want a `get_obect_for_user_or_404` helper?

Reply:
I was thinking about that too

Reply:
Added in latest commit